### PR TITLE
FHAC-987: Regression Cleanup for DocQueryFanouttest project

### PR DESCRIPTION
**Pull request by sailajaa**
1. Removed "update config", "Generate patient ID for Doc Query" scripts that are not required for all test cases.
2. Corrected correlation steps.
3. Organized script and removed log.info entries.
4. Removed uddiConnectionInfoMixSpecVersion.xml which is not required anymore.

### CODE REVIEW

**Comment by sailajaa** [comment]
@mhpnguyen @danielrf123 : PR assigned for review.

File: Product/SoapUI_Test/RegressionSuite/Standard/DocQueryFanoutTestTarget/DocQueryFanoutTestTarget-soapui-project.xml [hunk 1]
```diff
@@ -781,63 +737,36 @@ nhinc.FileUtils.createOrUpdateConnection("uddiConnectionInfo.xml", destConfigFil
          <urn:NhinTargetCommunities>
             <urn3:nhinTargetCommunity>
                <urn3:homeCommunity>
-                  <urn3:description>${#Project#UDDIhc1description}</urn3:description>
-                  <urn3:homeCommunityId>${#Project#UDDIhcid1}</urn3:homeCommunityId>
-                  <urn3:name>${#Project#UDDIhc1description}</urn3:name>
+                  <urn3:description>${#Project#UDDIhc3description}</urn3:description>
```
Comment (mhpnguyen) [inline Product/SoapUI_Test/RegressionSuite/Standard/DocQueryFanoutTestTarget/DocQueryFanoutTestTarget-soapui-project.xml:740]: Is this misspell? we have UDDIdc3description but not UDDIhc3description

**Comment by danielrf123** [comment]
I am good with a merge for this PR.

**Comment by sailajaa** [comment]
@mhpnguyen : Addressed PR comment. Ready for Review.

**Comment by mhpnguyen** [comment]
Endorse for merging without comments

**Pull request merged**